File: sdk/python/ataj.py
import subprocess
import sys

class Ataj:
  @staticmethod
  def run(file: str):
    return subprocess.run(['./atajc', 'run', file], check=True)

  @staticmethod
  def deploy(clouds: list):
    return subprocess.run(['./atajc', 'deploy', '--multi-cloud', ','.join(clouds)], check=True)

  @staticmethod
  def test():
    return subprocess.run(['./atajc', 'test', '--tier', 'all'], check=True)

if __name__ == "__main__":
  Ataj.run(sys.argv[1])

File: sdk/go/ataj.go
package ataj

import (
  "os"
  "os/exec"
  "strings"
)

func Run(file string) error {
  cmd := exec.Command("./atajc", "run", file)
  cmd.Stdout = os.Stdout
  cmd.Stderr = os.Stderr
  return cmd.Run()
}

func Deploy(clouds []string) error {
  cmd := exec.Command("./atajc", "deploy", "--multi-cloud", strings.Join(clouds, ","))
  return cmd.Run()
}
